### PR TITLE
tests: add ethernet eve test

### PR DESCRIPTION
**Pull request by regit**
## Ticket

If your pull request is related to a Suricata ticket, please provide
the full URL to the ticket here so this pull request can monitor
changes to the ticket status:

Redmine ticket: https://redmine.openinfosecfoundation.org/issues/6405

### CODE REVIEW

**Comment by regit** [comment]
I've just added one commit to cover more protocol (http and fileinfo). I'm stopping there. Sorry for update on live MR.

**Comment by catenacyber** [comment]
Requires https://github.com/OISF/suricata/pull/9715

File: tests/ethernet-eve/suricata.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+%YAML 1.1
+---
+
+include: ../../etc/suricata-4.0.3.yaml
```
Comment (catenacyber) [inline tests/ethernet-eve/suricata.yaml:4]: Really suricata4 ?

**Review by catenacyber — CHANGES_REQUESTED** [review]
Could you use a pcap instead of pcapng Eric ?

File: tests/ethernet-eve/suricata.yaml (hunk 1 above)
Comment (regit) [reply tests/ethernet-eve/suricata.yaml:4]: That's what is in the etc directory
```
❯ ls etc/
classification.config  reference.config  suricata-3.1.2.yaml  suricata-4.0.3.yaml
```

**Comment by regit** [comment]
replaced by #1667

**Pull request closed without merging**